lib/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -73,76 +64,71 @@ public UserIdentityService(ConfigurationService configurationService) {
                 new DataStore<>(
                         this.configurationService.getEnvironmentVariable(
                                 USER_ISSUED_CREDENTIALS_TABLE_NAME),
-                        UserIssuedCredentialsItem.class,
+                        VcStoreItem.class,
                         DataStore.getClient(isRunningLocally),
                         isRunningLocally,
                         configurationService);
     }
 
     public UserIdentityService(
-            ConfigurationService configurationService,
-            DataStore<UserIssuedCredentialsItem> dataStore) {
+            ConfigurationService configurationService, DataStore<VcStoreItem> dataStore) {
         this.configurationService = configurationService;
         this.dataStore = dataStore;
     }
 
     public List<String> getUserIssuedCredentials(String userId) {
```

Comment:
We should probably rename this method, maybe
```suggestion
    public List<String> getVcItemCredentials(String userId) {
```

Reply:
I was intentionally leaving this one. It is badly named, but it returns the underlying raw VC jwt, not the VcStoreItem. I think we want to have a richer object that wraps the raw jwt, but that's not this PR.